mod_sample/controllers.py

```diff
@@ -172,6 +172,13 @@ def serve_file_download(file_name, sub_folder='', content_type='application/octe
 
 @mod_sample.route('/download/<sample_id>')
 def download_sample(sample_id):
+    """
+    Download sample if the sample exist
+
+    :param sample_id: numbers
+    :raise: error
+    :return: function above
```

Comment:
That's still incorrect, and totally useless if someone decides to add a function in between.

Reply:
I think it is better to delete these doc-blocks
as I rethink that this doc-blocks are self explaintory